src/Cake.Core/CommandLineArguments/CommandLineArgumentsParser.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Cake.Core.CommandLineArguments
+{
+    internal static class CommandLineArgumentsParser
+    {
+        public static IDictionary<string, string> Parse(string[] arguments)
```

Comment:
Let's leverage the parsing of the arguments that is already done for us via Spectre.Console, which is available inside of [CakeArguments](https://github.com/cake-build/cake/blob/v1.1.0/src/Cake.Core/CakeArguments.cs#L10), so we don't need to implement additional logic to parse the arguments

Reply:
Done